Tests/NIOSSLTests/NIOSSLIntegrationTest.swift

```diff
@@ -1664,9 +1664,13 @@ class NIOSSLIntegrationTest: XCTestCase {
             try clientChannel.writeInbound(buffer)
```

Comment:
minor: could we use  `XCTAssertThrowsError` here instead of the `do { ... } catch { ... }` pattern? I know that was pre-existing code but see #196

Reply:
Feedback addressed and tested on macOS and Linux.